src/pages/contact_us/index.tsx

```diff
@@ -39,19 +46,17 @@ const ContactUs = () => {
     return (
         <Layout>
             <SEO
-                title={localize('Contact Us | Online Trading | Deriv.com')}
-                description={localize(
-                    'Have any query or need assistance, check out the details to call our international help desk and our email ID. Feel free to contact us anytime!',
-                )}
+                title={localize(title_text)}
+                description={localize(query_text)}
                 has_organization_schema
             />
             <SectionContainer tablet={{ p: '40px 0' }}>
                 <HeroWrapper fd="column">
                     <Header align="center" as="h1" type="display-title">
-                        {localize('Contact us')}
+                        {localize(contact_us_text)}
                     </Header>
                     <StyledText align="center" size="var(--text-size-sm)">
-                        {localize("Got questions? Here's how to get answers.")}
+                        {localize(questions_text)}
```

Comment:
```suggestion
                        <Localize translate_text={questions_text} />
```

Reply:
@niloofar-deriv what is the benefit of this approach over the old one ? we should use it like this everywhere ? 🙏

Reply:
As I mentioned in the parent card this is really important, our components need to be attached to react life cycle, so basically using `localize` in most of places are not quite ok.